docs/en/getting-started/get-started-stack.asciidoc

```diff
@@ -43,18 +40,33 @@ data, such as logs or decoded network packets.
 
 [TIP]
 ==========
-You can run Elasticsearch on your own hardware, or use our
-https://www.elastic.co/cloud/elasticsearch-service[hosted Elasticsearch Service]
-on Elastic Cloud. The Elasticsearch Service is available on both AWS and GCP.
+You can run {es} on your own hardware, or use our
+https://www.elastic.co/cloud/elasticsearch-service[hosted {es} Service]
+on Elastic Cloud. The {es} Service is available on both AWS and GCP.
 {ess-trial}[Try out the
-Elasticsearch Service for free].
+{es} Service for free].
 ==========
 
 To download and install {es}, open a terminal window and use the commands that
-work with your system (<<deb, deb>> for Debian/Ubuntu, <<rpm, rpm>> for
-Redhat/Centos/Fedora, <<mac, mac>> or <<brew, brew>> for OS X, <<linux, linux>>
-for Linux, and <<win, win>> for Windows):
+work with your system:
+
+* <<deb, deb>> for Debian/Ubuntu
+* <<rpm, rpm>> for Redhat/Centos/Fedora
+* <<mac, mac>> or <<brew, brew>> for OS X
+* <<linux, linux>> for Linux
+* <<win, win>> for Windows
 
+When you start {es} for the first time, the following security configuration
+occurs automatically:
+
+* {ref}/configuring-stack-security.html#stack-security-certificates[Certificates and keys] for TLS are
+generated for the transport and HTTP layers.
+* The TLS configuration settings are written to `elasticsearch.yml`.
+* A password is generated for the `elastic` user.
+* An enrollment token is generated for {kib}.
```

Comment:
This is not true for deb and rpm, not sure how we can differentiate it here without making it too complicated though

Reply:
We can add a note that indicates the difference with a link to the `elasticsearch-create-enrollment-token` tool:

>**NOTE**: On `deb` and `rpm` installations, an enrollment token isn't generated for {kib}. To generate an enrollment token for {kib}, use the [`elasticsearch-create-enrollment-token`](https://www.elastic.co/guide/en/elasticsearch/reference/8.0/create-enrollment-token.html) tool.

Reply:
"isn't generated for {kib}." -> "isn't generated for {kib} on installation." is my only suggestion. ++ otherwise